infra/gcp/terraform/modules/oci-proxy/monitoring.tf

```diff
@@ -16,23 +16,23 @@ limitations under the License.
 
 resource "google_monitoring_notification_channel" "emails" {
   display_name = "k8s-infra-alerts@kubernetes.io"
-  project      = google_project.project.project_id
+  project      = var.project_id
   type         = "email"
   labels = {
     email_address = "k8s-infra-alerts@kubernetes.io"
   }
 }
 
 module "alerts" {
-  project_id         = google_project.project.project_id
+  project_id         = var.project_id
   source             = "../monitoring/uptime-alert"
   documentation_text = "${var.domain} is down"
   domain             = var.domain
 
   notification_channels = [
     # Manually created. Monitoring channels can't be created with Terraform.
     # See: https://github.com/hashicorp/terraform-provider-google/issues/1134
-    "${google_project.project.id}/notificationChannels/${var.notification_channel_id}",
+    "projects/${var.project_id}/notificationChannels/${var.notification_channel_id}",
```

Comment:
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_project#id

> an identifier for the resource with format `projects/{{project}}`

I've also confirmed this is the value with `terraform state show` + there's no `terraform plan` diff.

Reply:
In other references we're just passing the project to terraform resources and either format works